### ISSUE TITLE
can i preprocess the libri_mel160_subword5000 with the preprocess_libri script?

### ISSUE THREAD

**Issue by ArtemisZGL** [andi611/Self-Supervised-Speech-Pretraining-and-Representation-Learning#39]
I have the librispeech dataset, but when i download the feature from the google drive, it's slow.  So I wonder can I  process it with the original dataset? I'm a little confuse about the "subword" meaning for mel feature.

**Comment by ArtemisZGL** [comment]
Beside, I also want to know about the meaning of name of pretrain ckpt, such as  M9 and M6. Thanks.

**Comment by andi611** [comment]
> I have the librispeech dataset, but when i download the feature from the google drive, it's slow. So I wonder can I process it with the original dataset? 

If you want to use features other then fMLLR, yes you can use the `preprocess/preprocess_libri.py`. See the instructions in `README.md`.
If you want to use fMLLR, please refer to this [issue](https://github.com/andi611/Self-Supervised-Speech-Pretraining-and-Representation-Learning/issues/37). In short, you have to download the fMLLR from google drive if you want to use our pre-trained models.

>I'm a little confuse about the "subword" meaning for mel feature.

Please ignore this, it is just a naming issue in our early development stage. The transcriptions are also preprocessed in subword units, but they were never used and deprecated.

> Beside, I also want to know about the meaning of name of pretrain ckpt, such as M9 and M6. Thanks.
M9 means masking 9 frames, M6 for 6 frames.

Hope this helps.
Andy

**Comment by ArtemisZGL** [comment]
Thank you for the detailed explanation! It's really a great job!
